### DISCUSSION TITLE
New install on Wyze v3 - can't get RTSP working

### DISCUSSION THREAD

**Discussion by rw377** [gtxaspec/wz_mini_hacks#579] [Q&A] [answered]
I installed the latest release of wz_mini_hacks per the instruction, am able to SSH into the device no problem. I edited the config once SSH'd in, and enabled Go2RTC. Once I'm in the Go2RTC web server and try to view a stream, the streams are erroring out. I also tried the RTSP link via VLC Player and it doesn't work either.  The config for the streams in Go2RTC is the default one that is set up when enabling the module in wz_mini_hacks. 

Not sure what the problem is or where to look further for logs/errors. Any help would be appreciated, thank you.
Edit: Currently on 4.36.10.4054 / Plugin version 2.43.0.7

![image](https://github.com/gtxaspec/wz_mini_hacks/assets/75325891/a8b022b6-afd6-464c-92d0-d0693e3a8edc)
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/75325891/04c7e0d2-3d2c-4d8d-b28f-b653874cfdc7)
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/75325891/b13845fa-2382-4ef8-a3ad-6d7399875f6d)
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/75325891/f4c5c45f-b84b-4eb7-98c3-4744d667ac0c)
![image](https://github.com/gtxaspec/wz_mini_hacks/assets/75325891/9580e50b-666d-4d46-a523-c6b272bd1d23)

**Comment by gtxaspec** [comment, marked as answer]
The firmware version is too new

See https://github.com/gtxaspec/wz_mini_hacks/wiki/Firmware-Support

**Reply by rw377** [reply]
Thanks. Is there a way to downgrade firmware?

**Reply by vic4news** [reply]
Yes. Look at the end of this discussion 
https://github.com/gtxaspec/wz_mini_hacks/discussions/570#discussioncomment-6518276
